geoportal/geoportailv3_geoportal/static-ngeo/less/layout.scss

```diff
@@ -52,7 +52,7 @@ app-search {
 
 .footer {
   position: absolute;
-  bottom: 0;
+  bottom: 2px;
```

Comment:
Looks like a magic value that need some explanations…

Reply:
That's the only value (and way) to make it work. Sounds like a hack..